app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
```

Comment:
you can use the `click` library, it is more intuitive to use and develop the CLI